guides/wide-ep-lws/README.md

```diff
@@ -68,6 +68,8 @@ GKE and CoreWeave are tested Kubernetes providers for this well-lit path. You ca
 
 ### Deploy InferencePool
 
+This guide uses a custom release name `deepseek-r1` and values file `inferencepool.values.yaml`. For standard InferencePool installation instructions, see the [inferencepool recipe](../../recipes/inferencepool/README.md#install-inferencepool).
```

Comment:
Just some context, the wide-ep-lws example currently doesn't use the reusable recipes (but it will be great to migrate this guide to use the reusable recipes).

The wide-ep-lws uses a  inferencepool profile for P/D settings. So I would expect that we create a `inferencepool-pd.values.yaml` in the recipes, and reference that here.

If you are interested, feel free to send a PR to update the wide-ep example with the new pattern.